src-vue/src/views/SettingsView.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <div class="fc_settings__container">
+        <!-- Game folder location -->
+        <h3>Manage installation</h3>
+        <el-input
+            v-model="$store.state.game_path"
+            class="w-50 m-2"
+            placeholder="Choose installation folder"
+            @click="updateGamePath"
+        >
+            <template #prepend>
+                <el-button icon="Folder" @click="updateGamePath"/>
+            </template>
+        </el-input>
+        <h3>About:</h3>
+        UI design inspired by <el-link :underline="false" target="_blank" href="https://github.com/TFORevive/tforevive_launcher/" type="primary">TFORevive Launcher</el-link> (not yet public)
+    </div>
+</template>
+
+<script lang="ts">
+export default {
+    name: "SettingsView",
+    methods: {
+        updateGamePath() {
+            console.log('TODO: update path');
+        }
```

Comment:
I assume this gets called when clicking the folder icon?

You can probably re-use the code I have in current main

https://github.com/GeckoEidechse/FlightCore/blob/a25a4ad4fda8c84ed3fd355f9494b28025dd0bbc/src-ui/src/main.ts#L55-L89

which uses a Tauri API to open OS file-picker (set to folders only)


https://github.com/GeckoEidechse/FlightCore/blob/a25a4ad4fda8c84ed3fd355f9494b28025dd0bbc/src-ui/src/main.ts#L3

Reply:
Fixed in 6665621da87811d2f219b83b21fb705b538dca90